tests/unit/h/services/group_members_test.py

```diff
@@ -76,15 +76,13 @@ def test_it_adds_users_in_userids(self, factories, group_members_service):
     def test_it_does_not_remove_existing_members(
         self, factories, group_members_service
     ):
-        creator = factories.User()
-        group = factories.Group(creator=creator)
-        users = [factories.User(), factories.User()]
-        userids = [user.userid for user in users]
+        group = factories.Group()
+        existing_member = factories.User()
+        group.members.append(existing_member)
```

Comment:
The test is testing that `add_members()` doesn't remove pre-existing members frmo the group. It was relying on the fact that the group's creator is a member of the group and asserting that the creator doesn't get removed. But the test actually has nothing to do with the concept of "creator", so I've changed it to just add an existing member to the group and assert that this existing member is still in the group after the call.